reana_client/cli/workflow.py

```diff
@@ -692,8 +710,19 @@ def workflow_run(ctx, file, filenames, name, skip_validation,
 @add_access_token_options
 @click.pass_context
 def workflow_delete(ctx, workflow, all_runs, workspace,
-                    hard_delete, access_token):
-    """Delete a workflow run given the workflow name and run number."""
+                    hard_delete, access_token):  # noqa: D301
+    """Delete a workflow.
+
+    By default removes all cached information of the given workflow and hides
+    it from the workflow list. Workspaces of deleted workflows are accessible
+    to retrieve files. To remove the workspace pass --include-workspace flag.
+    By passing --include-all-runs flag all workflows runs will be deleted. The
+    --include-records flag will delete all workflow data from the database and
+    remove its workspace.
+
+    Example: \n
+    \t $ reana-client delete -w workflow.13
```

Comment:
myworkflow.42

and perhaps add one more example that would remove everything:

--inculde-.. --include-...